### PR TITLE
Add scoop installation method

### PR DESCRIPTION
**Pull request by sitiom**
Adding a reference here as I just added [`typioca`](https://github.com/ScoopInstaller/Extras/blob/master/bucket/typioca.json) to Scoop. Also categorized the package managers via OS (Windows, Linux, and macOS) for organization.

### CODE REVIEW

**Review by bloznelis — CHANGES_REQUESTED** [review]
First of all, thanks for adding typioca to Scoop.

I don't think that there is a need to categorize these by operating systems. For example you can use Homebrew on Linux too.

And order of methods should be AUR > Brew > Scoop > Go.

**Comment by sitiom** [comment]
> I don't think that there is a need to categorize these by operating systems. For example you can use Homebrew on Linux too.

Homebrew is definitely an exception, will separate that.

> And order of methods should be AUR > Brew > Scoop > Go.

There shouldn't be an "order" of methods to use; one can choose from the many available options.

**Comment by bloznelis** [comment]
I'm pretty sure users are aware of their operating system and only care about specific package manager support.

> There shouldn't be an "order" of methods to use; one can choose from the many available options.

There should be an order, which is by relevancy. Windows users usually are not very terminal oriented bunch (and this is completely subjective opinion of mine). So as most of the users will be probably on Linux/Mac, these systems' package managers should be higher up the list, so they could be found faster.

**Comment by sitiom** [comment]
> I'm pretty sure users are aware of their operating system and only care about specific package manager support.
> 
> > There shouldn't be an "order" of methods to use; one can choose from the many available options.
> 
> There should be an order, which is by relevancy. Windows users usually are not very terminal oriented bunch (and this is completely subjective opinion of mine). So as most of the users will be probably on Linux/Mac, these systems' package managers should be higher up the list, so they could be found faster.

This is what categorizing them is for, Linux/Mac users can find their package managers in their respective categories quickly. As support for more package managers appear, categorizing them can help organize the list and reduce clutter. Besides, they can also just use the search function.

**Comment by bloznelis** [comment]
> Updates on this? I have sorted the list alphabetically (except for go), so the order is Homebrew > Linux > Windows > Go. What do you think?

Having "For Linux and macOS users" doesn't make much sense even if we add additional package manger support, as there are not many supported for both OS'es. I'd say dropping the OS names and sorting in alphabetical order is good enough then.

**Comment by sitiom** [comment]
Done.

**Comment by bloznelis** [comment]
It's not sorted in alphabetical order now ?

**Comment by sitiom** [comment]
Ah right, fixed.

File: README.md [hunk 1]
```diff
@@ -20,19 +20,34 @@ Minimal, terminal based typing speed tester.
...
-1. `brew tap bloznelis/tap`
-2. `brew install typioca`
+```
+yay -S typioca-git
+```
 
 ### Go
-`go install github.com/bloznelis/typioca@latest`
 
-**Note:** This will install typioca in `$GOBIN`, which defaults to `$GOPATH/bin` or `$HOME/go/bin` if the GOPATH environment variable is not set.
```
Comment (bloznelis) [inline README.md:35]: This note should not be removed

**Comment by bloznelis** [comment]
Nice! Thank you for adding this to Scoop and your patience :clap:

**Pull request merged**